modules/command/common/handler_ids_cleaner_command.js

```diff
@@ -0,0 +1,57 @@
+const Command = require('../command');
+const Models = require('../../../models/index');
+const constants = require('../../constants');
+const path = require('path');
+const fs = require('fs');
+
+/**
+ * Increases approval for Bidding contract on blockchain
+ */
+class HandlerIdsCleanerCommand extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.logger = ctx.logger;
+    }
+
+    /**
+     * Executes command and produces one or more events
+     * @param command
+     */
+    async execute(command) {
+        const timeToBeDeleted = Date.now() - constants.HANDLER_IDS_COMMAND_CLEANUP_TIME_MILLS;
+        await Models.handler_ids.destroy({
+            where: {
+                timestamp: { [Models.Sequelize.Op.lt]: timeToBeDeleted },
+            },
+        });
```

Comment:
Since the command itself is 2 lines of code, I'd recommend we just add this part to the cleanerCommand

Reply:
We can't do that they have different period of execution